### PR TITLE
fix(file-uploader): incorporate audit feedback

### LINKED ISSUE #2088: File uploader: Remainder of audit fixes
From @laurenmrice at #2048:

- [ ] For the error state, the input text needs to be disabled-02 (#bebebe). Make sure the error message text is calling caption-01 (support-01, #da1e28). ~The warning icon is pulling in the old icon, but this seems to be a global issue for components.~ The icon is correct disregard this.

- [x] So we can show a success example but we need to pull in the 16/checkmark and it should be color token interactive-01 (#0062ff) https://carbon-elements.netlify.com/icons/examples/preview/#16%2Fcheckmark

- [x] Also make sure the loading spinner is pulling in the one that is being used on the inline loading page.

- [x] The close icon, checkmark icon and loader need to be 16px away from the right. 
File uploader - Specs - White theme Copy 2

### PR DESCRIPTION
**Pull request by emyarod**
Closes #2088

#### Changelog

**Changed**

- invalid file uploader item text color token is now `disabled-02`
- invalid form message is now calling `caption-01`
- success icon (`CheckmarkFilled16` to `Checkmark16`)
- success icon color (`support-02` to `interactive-01`)
- increased file uploader item right padding

### CODE REVIEW

**Review by joshblack — APPROVED** [review]

**Review by lovemecomputer — APPROVED** [review]

File: packages/components/tests/spec/file-uploader_spec.js [hunk 1]
```diff
@@ -53,16 +56,17 @@ describe('File Uploader', function() {
     it('should set default options', function() {
       expect(flattenOptions(instance.options)).toEqual({
         selectorInit: '[data-file]',
-        selectorInput: 'input[type="file"].bx--file-input',
+        selectorInput: `input[type="file"].${prefix}--file-input`,
```
Comment (asudoh) [inline packages/components/tests/spec/file-uploader_spec.js]: We don't do variable prefix in test at this moment, rather tests see if default prefix is applied.

**Comment by emyarod** [comment]
looking into increasing coverage for file uploader tests to bring us over the threshold again

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Vanilla:
- [x] Change checkmark to be 16/checkmark--filled. Fill:`interactive-01` and Check: `icon-03`
Screen Shot 2019-06-12 at 2 48 10 PM

- [x] Change loader to have a container of 16px but the actual icon will be 14px within the container. Doing this to have the same size icon for all states.

- [x] Make the checkmark state file not closeable. 
- [x] Error state needs to have these specs to match the drag and drop file uploader (ref: #2962 )

Spec:
Screen Shot 2019-06-12 at 2 51 10 PM

- [x] When adding a file from the add button, the file shows up without a close icon. The close icon needs to appear and should be `icon-02`. 
Screen Shot 2019-06-12 at 2 09 17 PM

- [x] Upon adding a file we want to show the transition of the loader, to the checkmark, to the close icon like the current implementation for the drag and drop file uploader so we can be consistent.

**Comment by emyarod** [comment]
will consolidate these changes with @asudoh's #2848

**Comment by emyarod** [comment]
closing in favor of upcoming drag and drop file uploader PR

**Comment by emyarod** [comment]
got it, I removed `parentNode` usage in the new feature branch https://github.com/emyarod/carbon/tree/drag-and-drop-file-uploader

**Pull request closed without merging**